physionet-django/user/templates/user/edit_emails.html

```diff
@@ -62,7 +62,7 @@ <h5 class="modal-title">Remove Email</h5>
 
 <br>
 <h4>Primary Email</h4>
-<p>The email address used for authentication, and email notifications.</p>
+<p>Please use an institutional/academic email address. This will be used for authentication, and email notifications.</p>
```

Comment:
We should probably acknowledge that not everyone will have an institutional/academic email address. We could say something like: 

"The email address used for authentication, and email notifications. We recommend using an institutional/academic email address where possible."

Reply:
sounds good, added the sentence as part of the edit emails section.